okcupyd/looking_for.py

```diff
@@ -31,7 +31,8 @@ class LookingFor(object):
     profile.
     """
 
-    _ages_re = re.compile(u'Ages ([0-9]{1,3})\u2013([0-9]{1,3})')
+    _ages_re = re.compile(u'\s*Ages ([0-9]{1,3})\u2013([0-9]{1,3})\s*')
```

Comment:
I don't know what you mean.  There are two ways Ages can show up, depending on whether the minimum age is equal to the maximum age or not.  Most of the time, _ages_re will be the one that matches.